scripts/init-submodules-no-riscv-tools-nolog.sh

```diff
@@ -65,10 +63,9 @@ git submodule update --init generators/sha3
 git submodule update --init generators/gemmini
 git -C generators/gemmini/ submodule update --init --recursive software/gemmini-rocc-tests
 
-git config --unset submodule.sims/firesim.update
 # Minimal non-recursive clone to initialize sbt dependencies
 git submodule update --init sims/firesim
-git config submodule.sims/firesim.update none
+git config --local submodule.sims/firesim.update none
```

Comment:
Is there a reason to keep the firesim submodule permanently masked?  I'm inclined to omit this to be consistent with the treatment of other submodules.

Reply:
I think it was mostly to stop people from having an out-of-control recursive update (given the circular dependency between CY <-> FireSim) if they accidentally called `git submodule update` themselves instead of using the script.

Reply:
Ah, yeah, that makes a lot of sense.

As an additional measure, `update = none` could be added directly to the firesim section in `.gitmodules` so it applies by default, not just after the init-submodules script runs.  Then only explicitly running `git submodule update --checkout sims/firesim` would bypass the setting.

Reply:
Ah, i didn't know you could do that. That does seems cleaner.

Reply:
This works in testing, but I'll leave that for a future PR.